doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -290,7 +290,10 @@ Wi-Fi samples
 Other samples
 -------------
 
-|no_changes_yet_note|
+* Added:
+
+  * CoreMark sample added with support for nrf52, nr53 and nrf54l DK targets.
+    Sample enables easy way to measure performance of Nordic SoCs by running EEMBC CoreMark benchmark.
```

Comment:
```suggestion
* Added the :ref:`coremark_sample` sample that demonstrates how to easily measure a performance of the supported SoCs by running the Embedded Microprocessor Benchmark Consortium (EEMBC) CoreMark benchmark. 
  Included support for the nRF52840 DK, nRF5340 DK, and nRF54L15 PDK.
```

@zycz Can we also add this line as a first entry in the "documentation section" (the last section of the changelog file):

`` * Added the :ref:`ug_coremark` page. ``